tests/main.rs

```diff
@@ -254,7 +254,7 @@ fn test_samples(path: &str) -> Result<(), JaktError> {
                 let mut exe_name = temp_dir();
                 exe_name.push(format!("output{}", uuid));
 
-                let status = Command::new("clang++")
+                let status = Command::new("clang++-11")
```

Comment:
we should probably leave these as `clang++` as not all systems will have `clang++11` (mine doesn't, for example)

Reply:
That is a local change I need to do to get the testing to work. Will undo.

Reply:
If you're using ubuntu, I'm a fan of this script to keep things up to date as versions bump up :D
https://gist.github.com/junkdog/70231d6953592cd6f27def59fe19e50d